subsys/bluetooth/mesh/Kconfig

```diff
@@ -644,6 +641,20 @@ config BT_MESH_RELAY_RETRANSMIT_INTERVAL
 	  messages, in milliseconds. Can be changed through runtime
 	  configuration.
 
+config BT_MESH_RELAY_BUF_COUNT
+	int "Number of advertising buffers for relayed messages"
+	default 32
+	range 1 256
+	help
+	  Number of advertising buffers available for messages to be relayed.
+	  The bigger value increases reliability of the mesh network. The
+	  smaller value reduces the message latency on the relay node, but
```

Comment:
Also wondering if this should be written relay node or Relay Node as mentioned a little bit above in the text (menuconfig BT_MESH_RELAY).